src/ui/marker.js

```diff
@@ -484,4 +510,61 @@ export default class Marker extends Evented {
     isDraggable() {
         return this._draggable;
     }
+
+    /**
+     * Sets the `rotate` property for pointing the marker in a particular direction
+     * @param {number} [newRotate=0] Sets the direction the marker is pointing
+     * @returns {Marker} `this`
+     */
+    setRotate(newRotate: number) {
```

Comment:
for consistency with the rest of the codebase this can be simply `rotate` and instead of `newRotate`.

Reply:
Changed all the getters and setters